### ISSUE TITLE
[Android Bootcamp] Project 1 - Review my SimpleToDo app

### ISSUE THREAD

**Issue by gyaneshpandey09** [gyaneshpandey09/SimpleToDo#1]
My app is complete, please review. (@codepathreview and @codepath) 
I have the basic pre-work done and have also done some optional features. More work could be done with color and images.

**Comment by nesquena** [comment]
Hi Gyanesh,

Looks good, this exercise is intended in part to give you an introduction to the general rhythm of this course. The course is entirely project-based with an app being assigned each week and then due the following week. Each project builds on the last to help each engineer  learn the practical elements of Android development and best practices as quickly as possible. We also do a code review for each submitted project once the program begins.

The next step is to continue working on [extensions to your todo app](http://courses.codepath.com/snippets/intro_to_android/prework#heading-5-extending-your-todo). With regards to extensions to implement, we are looking for developers with an interest in user experience and product development so be sure to focus on additional functional enhancements and/or improving the user interface of the app. Please ping us back for a review once you've added a few extensions.

Note that the class is full at this point, but we are happy to add you to the waitlist. For those not admitted as participants, we will offer access to applicants as a remote observer cohort that has access to the materials and is able to work on the projects at their own pace.

**Comment by gyaneshpandey09** [comment]
Hi (@codepathreview and @codepath),
I have already added the following features as a part of my initial submission:
(Suggested) Persist the todo items into SQLite instead of a text file
(Suggested) Improve style of the todo items in the list using a custom adapter
(Suggested) Add support for completion due dates for todo items (and display within listview item)

It was because of these features that it took me extra time to do the submission.

Regards,
Gyanesh.

**Comment by nesquena** [comment]
Yes, I see those now. Thanks and we'll be contacting you again soon as we finalize admissions for the class. For those that are not able to participate, we will grant access to an observer program where you will be able to follow along self-paced with other remote observers with access to content and project.

**Comment by gyaneshpandey09** [comment]
Hi Nathan,
Could you please provide me with the observer access to the material, as you mentioned in your earlier comments?

Thanks,
Gyanesh.

**Comment by nesquena** [comment]
Of course, will send out more details soon.
